### ISSUE TITLE
Typo in man page?

### ISSUE THREAD

**Issue by varmanishant** [deadpixi/sam#52]
https://github.com/deadpixi/sam/blob/master/doc/sam.1#L42

Is that `or.Dq` a typo?

I opened `sam` and then tried `B filename` in another terminal to see if I can load a file from outside. It executed without errors but I didn't see the file in the `samterm`. I couldn't understand what the man says clearly. Are there any pointers or examples?

I think create a FAQ will be a nice idea in future.

**Comment by deadpixi** [comment]
Yeah, that's a typo. I fixed it, thank you.

The whole work behind the B command is a hack (and, IMHO, has been for 30 years). The right way to do it would be to have the host part of sam handle external commands, but that's just not feasible with the way the code is right now.

So, the way the '-r' flag works is like this:

If I start sam as "sam -r remotehost", sam will SSH over to remotehost and start a sam host part over there, then start a samterm part locally.

On remotehost, I can type just "B foo" to open up the file "foo" in sam.

Locally, I can type "B -r remotehost foo" to open up the file "foo" on remotehost (the same file) in sam.

Basically, you only need toe -r flag to B if you're trying to talk to a sam instance not on the local machine.

I know it's confusing, and if I ever get enough free time to fix it, I'll come up with something better. Believe it or not, this is actually an improvement over how it was in classic sam: you just didn't get to use B on the remote machine. :)

**Comment by varmanishant** [comment]
Thank you for correcting that typo. I will this out!
